README.md

```diff
@@ -234,7 +234,7 @@ http.HandleFunc("/", instana.TracingHandlerFunc(sensor, "/", func(w http.Respons
 }))
 ```
 
-In case your handler is implemented as an http.Handler, pass its `ServeHTTP` method instead:
+In case your handler is implemented as a http.Handler, pass its `ServeHTTP` method instead:
```

Comment:
```suggestion
In case your handler is implemented as a `http.Handler`, pass its `ServeHTTP` method instead:
```
Let's apply some formatting since we're on it :)

Reply:
Agree 👍